go/registry/metrics.go

```diff
@@ -33,10 +37,17 @@ var (
 			Help: "Number of registry runtimes.",
 		},
 	)
-	registeryCollectors = []prometheus.Collector{
+	registryNodeRegistered = prometheus.NewGauge(
```

Comment:
This should probably be moved to the registration worker in `go/worker/registration` as that will trigger registration in the first place. No need to check every time whether the node is registered.